spec/services/users/form526_user_identifiers_status_service_spec.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Users::Form526UserIdentifiersStatusService do
+  describe '#call' do
+    let(:user) { build(:user) }
+
+    describe 'participant_id validation' do
+      context 'when participant_id is missing' do
+        it "returns a hash with 'participant_id' marked false" do
+          allow(user).to receive(:participant_id).and_return(nil)
```

Comment:
🗣️ Have you tried a different approach that didn't require stubbing out the messages? For example:

```ruby
RSpec.describe Users::Form526UserIdentifiersStatusService do
  describe '#call' do
    subject(:call_service) { described_class.call(user) }

    describe 'participant_id validation' do
      let(:user) { build(:user, participant_id:) }

      context 'when participant_id is missing' do
        let(:participant_id) { nil }

        it "returns a hash with 'participant_id' marked false" do
          result = call_service
          expect(result['participant_id']).to be_falsey
        end
      end

      context 'when participant_id is present' do
        let(:participant_id) { '8675309' }

        it "returns a hash with 'participant_id' marked true" do
          result = call_service
          expect(result['participant_id']).to be_truthy
        end
      end
    end
  end
end
```

Reply:
Thanks, good suggestion, this needs to go out soon but I'll keep this in mind for my next specs!